ex01.ex

```diff
@@ -60,7 +60,7 @@ defmodule Ex01 do
   # no explicit + operators in your function                          #
   #####################################################################
 
-  sum3a = your_anonymous_function(1, 2, 3)
+  sum3a = fn (val1, val2, val3) -> sum2b.(val1, val2) |> sum2b.(val3) end
```

Comment:
You could even take the pipeline one step further:

~~~ elixir
  sum3a = fn a, b, c -> a |> sum2a.(b) |> sum2a.(c) end
~~~

(no deduction, just fyi)